docs/ref/contrib/admin/index.txt

```diff
@@ -2806,6 +2806,11 @@ if you specifically wanted the admin view from the admin instance named
     >>> change_url = urlresolvers.reverse('admin:polls_choice_change',
     ...                                   args=(c.id,), current_app='custom')
 
+If you want to change the password of a user, the URL can be resolved by::
+
+    >>> urlresolvers.reverse('admin:auth_user_password_change', args=(1,))
+    u'/admin/auth/user/1/password/'
```

Comment:
I don't think we need the example, but remember no u'' prefix as our docs are written with Python 3 in mind.